docs/pages/html-tl-controlled.html

```diff
@@ -22,7 +22,7 @@ <h2>Syntax</h2>
 	<p>
 		<code-template-html>
 			<template>
-				<input tl-controlled tl-attrmap="attribute:reference" tl-trigger="eventName" />
+				<input tl-controlled tl-attrmap="input-attribute:host-attribute" tl-trigger="eventName" />
```

Comment:
updating the naming here, so we can be consistent with the new docs, and to hopefully be more clear.